src/styles/blocks/normalize.scss

```diff
@@ -0,0 +1,22 @@
+body {
+  @include font($size: 10px, $weight: 400, $color: $main-accent-color);
```

Comment:
Ensure that the `font` mixin and the `$main-accent-color` variable are defined and imported correctly. Missing definitions could lead to styling issues.